src/app/Scenes/Artwork/Components/ArtworkLotDetails/utils/formateLotDateTime.ts

```diff
@@ -0,0 +1,11 @@
+import { DateTime } from "luxon"
+
+export const formateLotDateTime = (endAt: string) => {
```

Comment:
Small typo here on the name of the function + file name
```suggestion
export const formatLotDateTime = (endAt: string) => {
```

Reply:
good catch. fixed